substrate/frame/revive/src/exec.rs

```diff
@@ -198,7 +198,12 @@ pub trait Ext: sealing::Sealed {
 	/// Execute code in the current frame.
 	///
 	/// Returns the code size of the called contract.
-	fn delegate_call(&mut self, code: H256, input_data: Vec<u8>) -> Result<(), ExecError>;
+	fn delegate_call(
+		&mut self,
+		gas_limit: Weight,
```

Comment:
What about the deposit limit?

Also I'm wondering if we have the deposit limit whether this should just be a flag. Akin to how static calls are realized.

Reply:
I am not sure either but if we need it can be added later. This is enough for now I think

Reply:
Or maybe I can just add it

Reply:
@xermicus deposit_limit is added